### PR TITLE
fix(ingest): kafka ingest task hand up with error bootstrap server

### PR DESCRIPTION
**Pull request by wangsaisai**
…server

## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/kafka.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def create(cls, config_dict: Dict, ctx: PipelineContext) -> "KafkaSource":
         return cls(config, ctx)
 
     def get_workunits(self) -> Iterable[MetadataWorkUnit]:
-        topics = self.consumer.list_topics().topics
+        topics = self.consumer.list_topics(timeout=3).topics
```
Comment (treff7es) [inline metadata-ingestion/src/datahub/ingestion/source/kafka.py:163]: Please, can you make this configurable with a default value?

**Comment by jjoyce0510** [comment]
Lint appears to be failing! Once it passes we are good to go. 

Curious, what is the case in which this list topics query is failing? I'm wondering if we need to consider a more generic timeout config property.

File: metadata-ingestion/src/datahub/configuration/kafka.py [hunk 2]
```diff
@@ -16,6 +16,10 @@ class _KafkaConnectionConfig(ConfigModel):
         description="Extra schema registry config serialized as JSON. These options will be passed into Kafka's SchemaRegistryClient. https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html?#schemaregistryclient",
     )
 
+    list_topics_timeout_seconds: int = Field(
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/configuration/kafka.py:19]: can we rename this to something generalized? 

`client_timeout_seconds`. We will use this for another upcoming use case in the kafka source!

**Review by jjoyce0510 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**